scripts/ci/gitlab/pipeline/zombienet.yml

```diff
@@ -3,7 +3,7 @@
 
 zombienet-tests-parachains-smoke-test:
   stage:                           zombienet
-  image:                           "${ZOMBIENET_IMAGE}"
+  image:                           "docker.io/paritytech/zombienet:v1.3.16"
```

Comment:
Can we define this version as global, in the .gitlab-ci.yml file?

Reply:
On it!